src/sass/variables.scss

```diff
@@ -0,0 +1,34 @@
+
+$color-primary: #18191f;
+$color-secondary:  #404040;
+$color-navLink-borderColor: #474a57;
```

Comment:
Hey @ciradu2204, could we improve the variable naming here? 

$color-primary: #18191f;
$color-secondary: #404040;

$color-link-border: #474a57;
$color-font: #1c2334;

$border-dark-grey: #263147; 
$border-pale-grey: #aaa; 
$border-light-grey: #888;

$color-select-focus: #222;
$color-box-shadow-secondary: rgba(59,153,252,0.7); 

$color-error: rgba(0, 0, 0, 0.8); 
$color-placeholder-text: rgba(0,0,0,0.5); 
$color-bin-icon-secondary: rgba(0, 0, 0, 0.9);

$color-box-shadow: rgba(64,64,64,0.6); 
$border-dotted-grey: rgba(24,25,31,0.4); 
$color-bin-icon-primary: #d6fcf7;

$beige: rgba(255, 189, 18, 0.4); 
$white: white; 
$light-grey: #eeeff4;
[ <--- $bg-yesButton:  ....  could you please remove it, it's a duplicate color--->]
$dark-yellow: #ffbd12; 
$red: #f95a2c; 
$dark-grey: #002f3c; 
$light-red: #ff9692; 
$grey: #8385aa; 
$light-red: #ff9692;
$light-yellow: #ffd465;
$light-green: #61e4c5;
$light-pink: #fff3f8; 
$pale-blue: rgba(97, 228, 197, 0.1); 

$font-base: code-saver, monospace;

Reply:
Thank you @katia-utochkina . I have updated the variables.